changelog/next/features/4934--unroll-records.md

```diff
@@ -0,0 +1,3 @@
+The `unroll` operator now works for record fields as well as lists,
+deduplicating every non-null field in the specified record field while keeping
+the surrounding event intact.
```

Comment:
```suggestion
The `unroll` operator now works for records as well as lists,
deduplicating every non-null field in the specified record field while keeping
the surrounding event intact.
```

And in what way does it do `deduplicating`? Also needs to be updated in general with the new `null` semantics, I assume.

Reply:
That should be "duplicating", that de- prefix is wrong.

Reply:
I rewrote this to make it clearer.